Package.swift

```diff
@@ -10,6 +10,7 @@ let package = Package(
         .macOS(.v10_13),
         .iOS(.v12),
         .tvOS(.v12),
+		.visionOS(.v1),
```

Comment:
Looks like this is indented too far. Maybe it's just a case of spaces not tabs or vice-versa though. Not a big deal, just looks weird on the PR.